src/components/v2/V2Create/tabs/ReviewDeployTab/FundingSummarySection.tsx

```diff
@@ -0,0 +1,220 @@
+import {
+  getDefaultFundAccessConstraint,
+  getUnsafeV2FundingCycleProperties,
+} from 'utils/v2/fundingCycle'
+import {
+  formatReservedRate,
+  MAX_DISTRIBUTION_LIMIT,
+  weightedAmount,
+} from 'utils/v2/math'
+import SplitList from 'components/v2/shared/SplitList'
+import { NetworkContext } from 'contexts/networkContext'
+import TooltipLabel from 'components/shared/TooltipLabel'
+
+import { V2FundingCycle } from 'models/v2/fundingCycle'
+import { parseEther } from 'ethers/lib/utils'
+
+import { rowGutter } from '.'
+
+import {
+  AllowMintingStatistic,
+  DiscountRateStatistic,
+  DistributionLimitStatistic,
+  DurationStatistic,
+  IssuanceRateStatistic,
+  PausePayStatistic,
+  ReconfigurationStatistic,
+  RedemptionRateStatistic,
+  ReservedTokensStatistic,
+} from './FundingAttributes'
+
+export default function FundingSummarySection() {
+  const { payoutGroupedSplits, reservedTokensGroupedSplits } = useAppSelector(
+    state => state.editingV2Project,
+  )
+
+  const { userAddress } = useContext(NetworkContext)
+
+  const fundingCycleData = useEditingV2FundingCycleDataSelector()
+  const fundAccessConstraints = useEditingV2FundAccessConstraintsSelector()
+
+  const fundingCycle: V2FundingCycle = {
+    ...fundingCycleData,
+    number: BigNumber.from(1),
+    configuration: BigNumber.from(0),
+    basedOn: BigNumber.from(0),
+    start: BigNumber.from(Date.now()).div(1000),
+    metadata: BigNumber.from(0),
+  }
+
+  const fundingCycleMetadata = useEditingV2FundingCycleMetadataSelector()
+
+  const fundAccessConstraint = getDefaultFundAccessConstraint(
+    fundAccessConstraints,
+  )
+
+  const currencyName = V2CurrencyName(
+    fundAccessConstraint?.distributionLimitCurrency.toNumber() as V2CurrencyOption,
+  )
+
+  const distributionLimit = fundAccessConstraint?.distributionLimit
+  const hasDistributionLimit =
+    distributionLimit && !distributionLimit.eq(MAX_DISTRIBUTION_LIMIT)
+
+  const unsafeFundingCycleProperties =
+    getUnsafeV2FundingCycleProperties(fundingCycle)
+
+  const duration = fundingCycle.duration
+  const hasDuration = duration && duration.gt(0)
+
+  const initialIssuanceRate =
+    formatWad(
+      weightedAmount(
+        fundingCycle?.weight,
+        fundingCycleMetadata?.reservedRate.toNumber(),
+        parseEther('1'),
+        'payer',
+      ),
+      {
+        precision: 0,
+      },
+    ) ?? ''
+  const formattedReservedRate = parseFloat(
+    formatReservedRate(fundingCycleMetadata?.reservedRate),
+  )
+
+  return (
+    <div style={{ marginTop: 20 }}>
+      <h2 style={{ marginBottom: 0 }}>
+        <Trans>Funding cycle details</Trans>
+      </h2>
+      <p style={{ marginBottom: 15 }}>
+        {hasDuration ? (
+          <Trans>
+            These settings will <strong>not</strong> be editable immediately
+            within a funding cycle. They can only be changed for{' '}
+            <strong>upcoming</strong> funding cycles according to your{' '}
```

Comment:
nit: according to the project's reconfiguration rules

Reply:
Love